22-discovery.ks

```diff
@@ -31,6 +37,16 @@ echo " * enabling required system services"
 systemctl enable ipmi.service
 systemctl enable foreman-proxy.service
 systemctl enable discovery-fetch-extensions.path
+systemctl enable discovery-menu.service
+
+# register service is started manually from discovery-menu
+systemctl disable discovery-register.service
+
+echo " * disabling some unused system services"
+systemctl disable ipmi.service
+
+echo " * open foreman-proxy port via firewalld"
+firewall-offline-cmd --zone=public --add-port=8443/tcp
```

Comment:
We were not running firewall by default, this enables it (also need to load TFTP modules above).